src/ccl/charts/charts-container.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+
+import { ChartService } from '../services/chart.service';
+import { Chart } from './chart.component';
+
+/*
+ * Charts container component
+ * Holds all the charts
+ */
+
+@Component({
+  selector: 'charts',
+  directives: [Chart],
+  styleUrls: [
+    './chart.style.css',
+  ],
+  template: `<div class="chart" *ngFor="let chart of chartList">
+                <chart [indicator]="chart" [chartData]="chartData"></chart>
+            </div>`
+})
+
+export class Charts extends OnInit {
+
+    private chartList: Array<String>;
```

Comment:
I also imagined chartList eventually being an array of objects, but it's not there yet. When data is able to be pulled in by indicator from the api then it will be time.